### PR TITLE
handle cased messages

### PR DESCRIPTION
**Pull request by Daniel-Pivonka**

### CODE REVIEW

File: timbot.py [hunk 1]
```diff
@@ -55,20 +54,13 @@ def run_timbot():
...
 				message = message.encode('UTF8')
-
-				# openstack meme
-				if 'openstack' in message:
-					send_message('i hear opensack is a career killer')
-
-				# pong responce
-				elif 'pong' in message:
-					send_message('im in. best 2 out of 3 games to 7?')
+				message = message.lower()
```
Comment (timflannagan) [inline timbot.py]: does this work (line 56):
```python
message = message['text'].encode('UTF8').lower()
```

Comment (Daniel-Pivonka) [reply timbot.py]: ya that works

File: timbot.py [hunk 2]
```diff
@@ -54,21 +53,13 @@ def run_timbot():
...
-				elif 'pong' in message:
-					send_message('im in. best 2 out of 3 games to 7?')
+				message = message.encode('UTF8').lower()
 
 				if message.startswith(timbot_user_id):
 
 					# message now equals what was after '@timbot'
 					message = message.replace(timbot_user_id, '')
-
+
```
Comment (nathan-weinberg) [inline timbot.py]: Can we remove this whitespace?

File: timbot.py [hunk 3]
```diff
@@ -107,5 +98,13 @@ def run_timbot():
 					else:
 						send_message('keep pounding')
 
+				# openstack meme                     #temp fix
```
Comment (nathan-weinberg) [inline timbot.py]: Nested comments?

**Review by nathan-weinberg — CHANGES_REQUESTED** [review]

File: timbot.py (hunk 2 above)
Comment (timflannagan) [reply timbot.py]: Yeah dan get a linter please.

File: timbot.py (hunk 3 above)
Comment (Daniel-Pivonka) [reply timbot.py]: the comment is nested because it only applies to the second half of the elif this will be removed with #34

File: timbot.py [hunk 4]
```diff
@@ -7,7 +7,7 @@
 
 slack_token = os.environ['SLACK_API_TOKEN']
 channel_name = os.environ['SLACK_CHANNEL_NAME']
-timbot_user_id = os.environ['SLACK_TIMBOT_USER_ID']
+timbot_user_id = os.environ['SLACK_TIMBOT_USER_ID'].lower()
```
Comment (nathan-weinberg) [inline timbot.py:10]: Why are we lowering the user id?

Comment (Daniel-Pivonka) [reply timbot.py:10]: because i lowered the message and the message contains the id. so if left capitalized it will not be matched during the check

Comment (nathan-weinberg) [reply timbot.py:10]: bet

**Comment by Daniel-Pivonka** [comment]
rebased please review @nathan-weinberg @timflannagan1

**Review by nathan-weinberg — APPROVED** [review]
LGTM

**Pull request merged**